proto/forch_configuration.proto

```diff
@@ -165,6 +165,15 @@ message EventClientConfig {
  * Varz interface section in Forch configuration
  */
 message VarzInterfaceConfig {
-  // port number to curl for varz list
-  int32 varz_port = 1;
+  // port number to curl for forch varz list
+  int32 forch_varz_port = 1;
```

Comment:
this should be a generic map configuraiton, not specific to forch/faucet/gauge.  E.e., the config should have something like:

proxy_requests:
  proxy_port: 8032
  targets:
    faucet:
      port: 9032
    forch:
      port: 9036

Reply:
Addressed.